eng/pipelines/source-build-sdk-diff-tests.yml

```diff
@@ -0,0 +1,56 @@
+schedules:
+- cron: "0 7 * * 1-5"
+  displayName: Run on weekdays at 7am UTC
+  branches:
+    include:
+    - main
+    - release/*
+
+pr: none
+trigger: none
+
+pool:
+  name: NetCore1ESPool-Svc-Internal
+  demands: ImageOverride -equals 1es-ubuntu-2004
+
+parameters:
+- name: dotnetDotnetRunId
+  displayName: 'Specific dotnet-dotnet run ID number (e.g `2108850`)'
+  type: string
+  default: ' '
+
+jobs:
+- template: templates/jobs/sdk-diff-tests.yml
+  parameters:
+    buildName: CentOSStream8_Offline

```

Comment:
FYI, we have renamed the build leg names in https://github.com/dotnet/installer/pull/15603 for more clarity. It looks like you'll need to change these to match when you pull in the latest changes from main.

Reply:
I renamed them